packages/hello-world-react/app/pages/home/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
```

Comment:
I noticed some files missing the Copyright header.

Also here:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/84ba9c643f9df62e08c4420f646774a12b2ced4b/packages/hello-world-react/app/loader.js


Should we update the eslint rules or at very least include the header in these files?